fe/fe-core/src/main/java/org/apache/doris/master/Checkpoint.java

```diff
@@ -148,7 +148,15 @@ public synchronized void doCheckpoint() throws CheckpointException {
             // If failed, just return
             env = Env.getCurrentEnv();
             createStaticFieldForCkpt();
-            env.loadImage(imageDir);
+            try {
+                env.loadImage(imageDir);
+            } catch (Throwable e) {
+                LOG.error("Exception when check image validity", e);
+                if (MetricRepo.isInit) {
+                    MetricRepo.COUNTER_IMAGE_WRITE_INVALIDITY.increase(1L);
+                }
+                throw e;
+            }
```

Comment:
There is already a metric named `COUNTER_IMAGE_WRITE_FAILED`，why add another named `COUNTER_IMAGE_WRITE_INVALIDITY`